man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -273,3 +271,7 @@ The following exit values are returned:
     UEFI.  A hand-partitioned GPT disk may be able to be bootable with both
     BIOS and UEFI, and can have some of its other GPT parititions used for
     other purposes.
+
+    If a bootable pool's boot image or platform image becomes corrupt, even
+    if it's `zones`, a machine can still be booted with a USB stick, CD-ROM,
+    or other method of booting SmartOS, and recovery methods can occur.
```

Comment:
'recovery methods can occur' sounds a bit like recovery can spontaneously happen sometimes. Maybe reword it a bit?

Reply:
Will fix.